### PR TITLE
feat: Update json plugin for deephaven plugin refactor

### PR DESCRIPTION
**Pull request by devinrsmith**

### CODE REVIEW

**Comment by jnumainville** [comment]
We're using the black code formatter now, through a pre-commit hook for easy use - https://github.com/deephaven/deephaven-plugins#development
That said, just running black directly in this case might be more convenient

**Comment by devinrsmith** [comment]
> We're using the black code formatter now, through a pre-commit hook for easy use - https://github.com/deephaven/deephaven-plugins#development That said, just running black directly in this case might be more convenient

It might be better to hook up _git_ hooks? You could automate the venv or just call `pre-commit` and link to how to set it up if it wasn't already. https://github.com/deephaven/demo-app/tree/main/git-hooks

**Comment by devinrsmith** [comment]
> The `pre-commit install` step does set up a git hook, but maybe we could simplify those steps further

Ah, okay. That seems okay. Haven't used that tool before, looks neat.

**Review by jnumainville — APPROVED** [review]

**Review by mofojed — CHANGES_REQUESTED** [review]

File: plugins/json/.flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
```
[the remark this answers is not in the document]
Comment (devinrsmith) [reply plugins/json/.flake8:1]: flake8 is a linter, while black is a formatter. They can be used in combination, but they do serve separate purposes. https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8.

If we want to remove flake8 from here, we should do that as a separate PR IMO.

**Review by mofojed — APPROVED** [review]

**Pull request merged**